editor/lib/display/header_view.dart

```diff
@@ -133,7 +134,9 @@ class _HeaderViewState extends State<HeaderView> {
       var textureName = widget.controller.layers[i].configs.textureFileName;
 
       if (!texures.containsKey(textureName)) {
-        texures[textureName] = widget.controller.layers[i].textureBytes!;
+        texures[textureName] =
+            (widget.controller.layers[i] as ParticularEditorLayer)
```

Comment:
I didn't realize why we need to have an abstraction for `ParticularLayer` and `ParticularEditorLayer`?

Reply:
For holding away unnecessary texture bytes from plugin (memory usage)

Reply:
Ahh okay, you just defined the ParticularEditorLayer inside the `/editor` app. I thought it is inside the plugin.

So it seems good to me.